makefile.unix

```diff
@@ -15,15 +15,20 @@
 #
 # make -f makefile.unix CFLAGS="-O3 -DUSE_LTM -DLTM_DESC -I/path/to/libtommath" EXTRALIBS=/path/to/libtommath/libtommath.a all
 # ./test
-# make -f makefile.unix DESTDIR=/opt/libtom install
+# make -f makefile.unix PREFIX=/opt/libtom install
 #
 #Or if you are using Intel C compiler you might need something like:
 #
 # make -f makefile.unix CC=icc AR=xiar CFLAGS="-fast -DUSE_LTM -DLTM_DESC -I/path/to/libtommath" EXTRALIBS=/path/to/libtommath/libtommath.a all
 #
 
 #The following can be overridden from command line e.g. "make -f makefile.unix CC=gcc ARFLAGS=rcs"
-DESTDIR   = /usr/local
+DESTDIR   =
+PREFIX    = /usr/local
+LIBPATH   = $(DESTDIR)$(PREFIX)/lib
```

Comment:
Ah, I forgot to remove that again... or do you think we should use `LIBPATH` & co. in all makefiles?